assets/src/application/ui/layout/confirm/ConfirmClose.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Popconfirm } from 'antd';
+import { PopconfirmProps } from 'antd/lib/popconfirm';
+import { RenderFunction } from 'antd/lib/_util/getRenderPropValue';
+import { __ } from '@wordpress/i18n';
+
+interface ConfirmDeleteProps extends Omit<PopconfirmProps, 'title'> {
+	title?: React.ReactNode | RenderFunction;
+}
+
+const ConfirmClose: React.FC<ConfirmDeleteProps> = ({ children, ...props }) => {
+	const cancelText = props.cancelText || __('No');
+	const okText = props.okText || __('Yes');
+	const title = props.title || __('Are you sure you want to delete this?');
```

Comment:
title and interface looks like copypasta from the ConfirmDelete component

Reply:
updated and extracted common types: https://github.com/eventespresso/event-espresso-core/pull/2636/commits/0ffc8e75222904aa56054094c64d9d089bbe2500